lib/hydra/works.rb

```diff
@@ -64,21 +64,38 @@ module Works
     autoload :GenerateThumbnail,                   'hydra/works/services/generic_file/generate/thumbnail'
     autoload :PersistDerivative,                   'hydra/works/services/generic_file/persist_derivative'
 
-    # model validations
-    def self.collection? collection
+    # model validations 
+    def self.works_collection? collection
```

Comment:
I would expect that test:

```ruby
expect( pcdm_coll.works_collection? ).to be false
```

to raise a NoMethodError -- so maybe the more appropriate test for non-Hydra::Works objects would be something like:

```ruby
expect( pcdm_coll.respond_to? :works_collection ).to be false
```